README.md

```diff
@@ -28,22 +28,24 @@ Feedback, suggestions, questions or concerns? Open a new [GitHub issue](https://
 Some notable features of the wp-parsely plugin are:
 
 - Automatically inserts the Parse.ly metadata and JavaScript in all published pages and posts (supports Custom Post Types).
-- [Supports Google Tag Manager, AMP, Facebook Instant Articles, Google Web Stories and Cloudflare](https://www.parse.ly/help/integration/frequently-asked-questions#third-party-and-feature-support).
-- Offers the `wpParselyOnLoad` and `wpParselyOnReady` JavaScript hooks that allow advanced integrations requiring JavaScript, such as [Dynamic Tracking](https://www.parse.ly/help/integration/dynamic-tracking).
-- [Supports WordPress Network (Multisite) setups](https://www.parse.ly/help/integration/frequently-asked-questions#third-party-and-feature-support).
-- [Supports decoupled (headless) setups](https://www.parse.ly/help/integration/decoupled-headless-support).
-- Provides a [Recommendations Block](https://www.parse.ly/help/integration/recommendations-block) that shows a list of links related to the currently viewed page. Useful for showcasing related content to visitors.
-- Provides a [Content Helper](https://www.parse.ly/help/integration/content-helper) in the WordPress Editor sidebar that displays top-performing posts based on the currently edited post's tag or category. Useful for editors to see which related content is popular so they can gain insights from it.
+- [Supports Google Tag Manager, AMP, Facebook Instant Articles, Google Web Stories and Cloudflare](https://docs.parse.ly/plugin-common-questions/#h-is-wp-parsely-compatible-with-amp-facebook-instant-articles-or-google-web-stories).
+- Offers the `wpParselyOnLoad` and `wpParselyOnReady` JavaScript hooks that allow advanced integrations requiring JavaScript, such as [Dynamic Tracking](https://docs.parse.ly/plugin-dynamic-tracking/).
+- [Supports WordPress Network (Multisite) setups](https://docs.parse.ly/plugin-common-questions/#h-is-wp-parsely-compatible-with-wordpress-network-multisite).
+- [Supports decoupled (headless) setups](https://docs.parse.ly/plugin-decoupled-headless-support/).
+- Provides a [Recommendations Block](https://docs.parse.ly/recommendations-block/) that shows a list of links related to the currently viewed page. Useful for showcasing related content to visitors.
+- Provides a [Content Helper](https://docs.parse.ly/plugin-content-helper/) in the WordPress Editor sidebar that displays the following panels:
```

Comment:
When are we going to update content or screenshots on this page?

Reply:
Upon release I'll update the Content Helper docs page and see where new screenshots are needed.